### ISSUE TITLE
script with two equal subcommand

### ISSUE THREAD

**Issue by bkoshelev** [lamartire/sharec#142]
**Describe the bug**

Sharec rewrite script, if command have equal **subcommand**

![image](https://user-images.githubusercontent.com/2634710/89707449-a2dd4580-d976-11ea-980b-b12f791d1412.png)

**Steps to reproduce the behavior**

1. Add to scripts:

`    
"build:local": "next build && next export",
`

2. reinstall sharec in project

**Expected behavior**

`    "build:local": "next build && next export",
`

**Code snippets**

`node_modules/sharec/src/strategies/helpers/params/commandsToMap.js
`

```
      parsedCommand.set(subcommand, parsedSubcommand) //subcommand = "next"
```

**System**

sharec@2.5.0

**Comment by lamartire** [comment]
Have you changed `build:local` script since the last configuration update?

**Comment by bkoshelev** [comment]
> Have you changed `build:local` script since the last configuration update?

no

**Comment by lamartire** [comment]
Ok, I will see :ok_hand:

**Comment by lamartire** [comment]
So, finally I got, what you meant. Give me a time, to fix this problem.

**Comment by lamartire** [comment]
@bkoshelev solved in `2.5.1`
